.env-wb-db-event-listener

```diff
@@ -0,0 +1,38 @@
+#
+# Explicit plugins DISABLED in the webserver to create a garbage-collector
+# Docs plugins config of services/web/server/src/simcore_service_webserver/application_settings.py
+#
+
+
+WEBSERVER_ACTIVITY=null
+WEBSERVER_CATALOG=null
+WEBSERVER_NOTIFICATIONS=0
+WEBSERVER_DB_LISTENER=1

```

Comment:
So this is the main active plugin! Can you highlight it with a comment?

Reply:
ok fixed.